### ISSUE TITLE
Grafana "Node Count" chart shows incorrect Meta node count

### ISSUE THREAD

**Issue by rotten** [risingwavelabs/risingwave#15869]
### Describe the bug

In the Grafana dashboard, the `dev dashboard/Cluster Node/Node Count` chart shows incorrect values for the meta node.  We have two clusters running at the moment, Neither has never had more than 2 meta nodes running for the life of the cluster, and I don't think ever more than 1 at a time.  Currently they both only have **1** node running.  One system shows a node count of **5** and the other shows a node count of **20**.

### Error message/log

```text
There aren't any error messages that I've seen related to this.
```

### To Reproduce

Run a cluster for a while and verify the Node Count is correct.

### Expected behavior

I was expecting to see a single node on the chart.

### How did you deploy RisingWave?

Docker on AWS ECS

### The version of RisingWave

 PostgreSQL 9.5.0-RisingWave-1.7.2 (6ecc083c72b83a96e7e3b58ca3a19b28cd5f74bf)

**Comment by rotten** [comment]
I should add that the other node counts seem correct.

**Comment by fuyufjh** [comment]
Thanks for the feedback. Let me take a look

**Comment by rotten** [comment]
To follow up.  This morning I noticed that our cluster had been up 5 days and we were still running the original meta-node.  The dashboard indicated we only had one running.  So I killed the meta node.  A new one replaced it.  I killed it again, and a new one replaced that.  We are currently still only running 1 meta node.  However when I look at the node count in the Grafana dashboard, it now thinks we are running 3 meta nodes.
